### PR TITLE
feat(3000): Show the 3000 index-less navbar with labels

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

We want to initially release the 3000 look with labels in the navbar, not too dissimilar to what the app looks like now, so that the change isn't disorienting. This PRs add the labels, along with a few navbar items that were missing.

Note: when the `posthog-3000-nav` flag is active (i.e. sidebar-based navigation), the buttons are still icons.

### CODE REVIEW

File: frontend/src/layout/navigation-3000/components/Navbar.tsx [hunk 1]
```diff
@@ -22,16 +23,17 @@ export function Navbar(): JSX.Element {
...
-            <div className="Navbar3000__content">
-                <div className="Navbar3000__top">
-                    {navbarItems.map((section, index) => (
-                        <ul key={index}>
-                            {section.map((item) => (
+            <div className="Navbar3000__top">
+                {navbarItems.map((section, index) => (
+                    <ul key={index}>
+                        {section.map((item) =>
+                            item.featureFlag && !featureFlags[item.featureFlag] ? null : (
```
Comment (benjackwhite) [inline frontend/src/layout/navigation-3000/components/Navbar.tsx:36]: Nice `featureFlag` helper!

**Review by benjackwhite — APPROVED** [review]
Love it!

We discussed perhaps trying out the resizer thing in a follow up once this and https://github.com/PostHog/posthog/pull/18204 are merged 👍

**Comment by corywatilo** [comment]
A few suggestions:

- Replace "Project homepage" with the actual project name
- Persons and groups -> People
- Just move Cohorts into People now
- Data management -> Data
- Just move Annotations into Data now
- Support beta/alpha labels?
- The second section is intended for products, so I'd move Toolbar out to the lower section
- Alphabetize products
- Project settings -> Settings

If we make these changes now, we'll be pretty close to what we'll release in full 3000, so we can just have a tooltip/popup saying "We moved some things around! Data is up top, products are in the middle. Cohorts now live in Data, and Cohorts moved to People"

**Comment by Twixes** [comment]
Thanks for considering this! @corywatilo
Tried addressing all your points. With three I ran into some blocking trickiness (marked with orange):
1. ➡️ I think that makes sense, though it should happen alongside the project name being pulled out of the breadcrumbs bar, so will do that in a separate PR.
2. ❇️ Agreed with "People", but a bit wary of removing "groups" from that label, because 1. makes it harder to learn about group analytics 2. in our model groups are not made up of people – they're made up purely out of events. Named "People and groups" for now.
3. ➡️ Cohorts and annotations are being moved in #18200.
4. 🆚 I don't quite see renaming "Data management" to "Data" improving clarity. I mean, "Data" is more concrete, but it's also concretely _not_ what's on that page. The data can be found in "People and groups", and "Event explorer". "Data management" is basically "Data taxonomy" though – _perhaps_ that'd be clearer?
5. ➡️ Same as 3.
6. ✅ Good catch, added alpha and beta labels. They feel too prominent in dark mode, but that's a color scheme issue…
     I came up with an outlined style for that case, but it's not legible in light mode, and not sure if it'll work outside of the navbar in dark. Hence leaving out of this PR.
    Screenshot 2023-11-02 at 20 03 10

8. ✅ Put Toolbar in the top segment.
9. 🆚 One problem I see with alphabetic sorting is that related products would be far apart, e.g. "Product analytics" would be separated from "Web analytics" by "Session replay" + "Surveys". I don't feel like this is outweighed by the alphabetic aspect.
10. ✴️ The "Project" part in "Project settings" is currently pretty key, because a user might easily think that's where you can find all the settings or at least navigate wherever you need. That's not the case, there are separate entry points (i.e. buttons) for organization and account settings. It would be nice to be able to treat this button as just "Settings", but currently we can't – we'll need to solve this as part of designing settings at some point (as the current experience is the result of pure organic growth).

### Current state:

**Comment by corywatilo** [comment]
> Settings

Just FYI I _do_ have some thoughts on the settings UI [here](https://github.com/PostHog/posthog/issues/12923#issuecomment-1435104201).

**Comment by Twixes** [comment]
Deployed this, but obviously happy to follow up on all the remaining (orange) points! Settings very much included.

**Pull request merged**